src/hardware/developing-an-iox/protocol.md

```diff
@@ -281,6 +281,18 @@ Sent from the IOX to the GO device to inform the GO device of events or status c
 | 0-1 | 0x0004 |
 | 2 | Unused |
 
+#### Information Type 11 IOX Third-Party Tx-Buffer State (GO9+ only)
```

Comment:
Suggest changing the wording of GO9+ to (GO9 and newer only). We know what GO9+ is referring to internally, but there is actually a product called GO9+. It is a GO9 with WiFi.

Reply:
Fixed.

Reply:
Renamed the information type to: 

> Information Type 11 - IOX Stop/Resume GO Tx messages (GO9 and newer only)